src/libraries/System.Private.Xml/src/System/Xml/Dom/XmlNodeReader.cs

```diff
@@ -897,7 +897,7 @@ internal string DefaultLookupNamespace(string prefix)
                 {
                     return _nameTable.Add(XmlReservedNs.NsXml);
                 }
-                if (prefix == string.Empty)
+                if (prefix.Length == 0)
```

Comment:
What guarantees prefix is non-null here?

Reply:
Well, I think nothing guarantees that.
Should it be `string.IsNullOrEmpty` then? or reverted back to `prefex == string.Empty`?
I don't know what should happen if it was null.

Reply:
```suggestion
                if (prefix == string.Empty)
```